host/greatfet/commands/greatfet_chipcon.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+#
+# This file is part of GreatFET
+
+from __future__ import print_function
+
+import errno
+import sys
+import ast
+
+import greatfet
+from greatfet import GreatFET
+from greatfet.utils import log_silent, log_verbose
+from greatfet.programmers.chipcon import ChipconProgrammer
+
+
+def main():
+    from greatfet.utils import GreatFETArgumentParser
+
+    # Set up a simple argument parser.
+    parser = GreatFETArgumentParser(description="Utility for chipcon debugging via GreatFET")
+    parser.add_argument('-id', '--chip_id', action='store_true',
```

Comment:
Short options (one dash) should always be one letter, and multiple words in long options are conventionally separated by dashes, not underscores. This goes for the rest of the options as well.